templates/search/results-no-results-en.html

```diff
@@ -0,0 +1,46 @@
+---
+{
+	"title": "Search results (no results)",
+	"language": "en",
+	"altLangPage": "results-no-results-fr.html",
+	"pageType": "search",
+	"pageclass": "page-type-search",
+	"dateModified": "2025-07-25",
+	"share": "false",
+	"deptfeature": false,
+	"nositesearch": true
+}
+---
+<form role="search">
+	<label class="wb-inv" for="sch-inp">Search Government of Canada websites</label>
+	<div class="input-group mrgn-tp-lg">
+		<input spellcheck="false" autocomplete="off" id="sch-inp" class="form-control" type="search" data-fusion-query name="q" aria-describedby="gc-pi">
+		<span class="input-group-btn">
+			<button type="submit" class="btn btn-primary"> <span class="glyphicon-search glyphicon"></span><span class="wb-inv">Search</span></button>
+		</span>
+	</div>
+	<p id="gc-pi" class="mrgn-tp-md">Don't include personal information (telephone, email, SIN, financial, medical, or work details).</p>
+</form>
+
+<div data-gc-search="{}">
+	<div id="wb-land">
```

Comment:
this should be a section

```suggestion
	<section id="wb-land">
```